roles/ceph-update/tasks/main.yml

```diff
@@ -0,0 +1,47 @@
+---
+- name: check if osds are up
+  shell: ceph osd stat | awk '{print $5}' | egrep -sq "$(ceph osd stat | awk '{print $3}')"
+  register: result
+  until: result.rc == 0
+  retries: 10
+  delay: 10
+  delegate_to: "{{ groups['ceph_monitors'][0] }}"
+
+- name: restart ceph mons
+  service: name=ceph-mon-all state=restarted
+
+# Setting noout to prevent cluster from rebalancing after service restart
+- name: set osd noout
+  command: ceph osd set noout
+  delegate_to: "{{ groups['ceph_monitors'][0] }}"
+  when: inventory_hostname in groups['ceph_osds'][0]
```

Comment:
Oh, yeah, you should probably change the `in` to `==` just to be consistent with the rest of this file, and with other such conditionals in Ursula (although I wouldn't be surprised if you found both styles)